pkg/libmpdclient

```diff
@@ -0,0 +1,26 @@
+[mirrors]
+https://musicpd.org/download/libmpdclient/2/libmpdclient-2.9.tar.xz
+
+[vars]
+filesize=253992
+sha512=7e6af51e31a3319e6681dfe5199fda1b2c1482f6aa6854af7c7f02c999fbbeae4f369c9b157abf95402c7f022d155109454b243549185ba791df5b2457baaba4
+pkgver=1
+
+[deps]
+patch
```

Comment:
are you adding patch as dependency since you apply a patch ? that's not necessary, since patch is hard required to install almost any package